dc

```diff
@@ -5,56 +5,59 @@
 # would use docker-compose with the only difference being that you don't need to specify compose files. For more
 # information you can run `./dc` or `./dc help`.
 
-if [ -f .env ]; then
+# get current working directory of this script
+cwd="$(dirname -- "$0";)";
+
+if [ -f ${cwd}/.env ]; then
   OLD_IFS=$IFS
   IFS=$'\n'
-  for x in $(grep -v '^#.*' .env); do export $x; done
+  for x in $(grep -v '^#.*' ${cwd}/.env); do export $x; done
```

Comment:
We need a comment explaining what exactly is happening here. I, personally, have no idea and future maintainers might not have either.

Reply:
added more info in https://github.com/SkynetLabs/skynet-webportal/pull/2202/commits/063f0b0b7bcf483553fa2dc88e6e351ea631598b